environment/Dockerfile

```diff
@@ -1,14 +1,22 @@
 # hash:sha256:4f567b195522ba414263860e03ae193d69e7c4114b288c652000066eab30db33
 ARG REGISTRY_HOST
-FROM $REGISTRY_HOST/codeocean/pytorch:2.4.0-cuda12.4.0-mambaforge24.5.0-0-python3.12.4-ubuntu22.04
+
+# --- Base image and Jax for CPU (preferred) ---
+FROM $REGISTRY_HOST/codeocean/mambaforge3:24.5.0-0-python3.12.4-ubuntu22.04
+ARG JAX_PKG="jax==0.6.1"
+
+# --- Base image and Jax for GPU ---
```

Comment:
Are users supposed to uncomment this block if they want to switch to GPU? If so, can we document that? 

Did you modify the dockerfile directly, or do this via the environment panel?